src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -100,7 +100,7 @@ class WorkspaceInvitePage extends React.Component {
 
     getExcludedUsers() {
         const policyEmployeeList = lodashGet(this.props, 'policy.employeeList', []);
-        return [...EXCLUDED_GROUP_EMAILS, ...policyEmployeeList];
```

Comment:
`EXCLUDED_GROUP_EMAILS` this is very similar to `EXPENSIFY_EMAILS`. But I'm having trouble understanding why we need this separate concept. To avoid confusion on the future maybe we should do the following:

1. Only have `EXPENSIFY_EMAILS` and not these other arrays here

https://github.com/Expensify/App/blob/d0abb528c4c7188c21dfe189c996ce00d750ce10/src/CONST.js#L457-L496

2. If we want to include additional emails to a blocklist then we can merge them with `EXPENSIFY_EMAILS`
3. If we want to allow an email in that array then we can omit it from the list

Maintaining three different lists seems error prone to me.

Alternatively, we could remove Concierge from the `EXPENSIFY_EMAILS` list and explicitly add them in any place where we also want them to be blocked. Again, not maintaining the separate lists.

Reply:
Ok. Then It will be delayed a bit. I will check the order parts of the app that are using the Exclude arrays and normalize them.

Reply:
I am thinking of just replacing the other two arrays with something like
```
EXCLUDED_GROUP_EMAILS = _.without(EXPENSIFY_EMAILS, [...])
```
It keeps them centralized instead of excluding the items on each page.

What do you think? @marcaaron.